client/pages/auth/register.jsx

```diff
@@ -37,7 +37,7 @@ const Register = () => {
       console.log('Registration successful:', responseData);
 
       // Redirect to another page or perform other actions after successful registration
-      //router.push('/');
+      router.push('/auth/login');
```

Comment:
Why are you redirecting user to login page after registration? 
Save the token and redirect them to the app